### PR TITLE
Use console.warn instead of console.error when we catch a failed fetch

### PR DESCRIPTION
**Pull request by eyeseast**
Tell Sentry these aren't as high a priority as other errors.

### CODE REVIEW

File: src/lib/api/tests/addons.test.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ describe("addon payloads", () => {
     const payload = addons.buildPayload(scraper, form, true);
 
     if (payload.errors) {
-      console.error(payload.errors);
+      console.warn(payload.errors);
```
Comment (allanlasser) [inline src/lib/api/tests/addons.test.ts:174]: Not sure if this should change, as it's in a test. Do we want to be logging in tests at all?

File: src/lib/components/viewer/PDF.svelte [hunk 2]
```diff
@@ -61,7 +61,7 @@
         window.pdf = p;
       })
       .catch((e) => {
-        console.error(e);
+        console.warn(e);
```
Comment (allanlasser) [inline src/lib/components/viewer/PDF.svelte:64]: Rendering PDFs is on the critical path—I think it's fair to treat these as elevated.

File: src/lib/components/viewer/ViewerContext.svelte [hunk 3]
```diff
@@ -173,7 +173,7 @@ layouts, stories, and tests.
       };
 
       task.promise.catch((error) => {
-        console.error(error);
+        console.warn(error);
```
Comment (allanlasser) [inline src/lib/components/viewer/ViewerContext.svelte:176]: Rendering PDFs is on the critical path. I think we want to keep this error elevated.

File: src/lib/api/tests/addons.test.ts (hunk 1 above)
Comment (eyeseast) [reply src/lib/api/tests/addons.test.ts:174]: I think I had that in so I could debug while working on validation. Don't think we need it now.

**Review by allanlasser — APPROVED** [review]

**Pull request merged**